app/src/main/java/com/anytypeio/anytype/ui/home/HomeScreenFragment.kt

```diff
@@ -245,6 +247,13 @@ class HomeScreenFragment : BaseComposeFragment() {
                     )
                 )
             }
+            is Command.Deeplink.MembershipScreen -> {
+                findNavController().navigate(
+                    R.id.paymentsScreen,
+                    MembershipFragment.args(command.tierId),
+                    NavOptions.Builder().setLaunchSingleTop(true).build()
```

Comment:
please add comment why single top is important here, so that we don't forget.

Reply:
there is no point in opening another Membership screen on top if suddenly Membership is already open